src/electron/platform/android/rule-information-provider.ts

```diff
@@ -92,8 +92,8 @@ export class RuleInformationProvider {
         );
     };
 
-    private buildUnifiedResolution(unformattedText: string, codeStrings: string[] = null): UnifiedResolution {
-        return { howToFixSummary: unformattedText, formatAsCode: codeStrings };
+    private buildUnifiedResolution(unformattedText: string, codeStrings: string[] = null): UnifiedFormatAsCodeResolution {
```

Comment:
howToFixFormat is not necessarily limited to formatting as code. It could include italics, links, or any other number of things. The type name should not be bound to "FormatAsCode". UnifiedFormattedResolution, perhaps?

Also, the rename should extend to include RuleInformation's GetUnifiedResolutionDelegate type and getUnifiedResolution method